apps/frontend/src/app/components/map-review/map-review-suggestions-form.component.ts

```diff
@@ -94,12 +94,8 @@ export class MapReviewSuggestionsFormComponent implements ControlValueAccessor {
           .map(({ trackNum }) => trackNum)
       );
 
-      for (let i = 0; bonusNums.size < 0; i++) {
-        if (!bonusNums.has(i)) {
-          trackNum = i;
-          break;
-        }
-      }
+      trackNum = 1;
```

Comment:
Is this assignment necessary? It's already assigned to 1 on line 84.

Reply:
Good spot, removing

Reply:
oh, this was a commit on a previous PR which I merged yesterday, whatever, will sneak a tiny commit into here